jaxrs-spec/src/main/asciidoc/chapters/appendix/_bibliography.adoc

```diff
@@ -36,25 +36,25 @@
 - [[[bib12,12]]]  RxJava 2.0: What’s different in 2.0. See
                https://github.com/ReactiveX/RxJava/wiki/What’s-different-in-2.0.
 
-- [[[bib13,13]]]  Anthony Lai. Concurrency Utilities for Java EE. JSR, JCP, March 2013. See
-               https://jcp.org/en/jsr/detail?id=236.
+- [[[bib13,13]]]  Jakarta™ Concurrency Specification, Version 1.1. Available at:
+               https://jakarta.ee/specifications/concurrency/1.1
 
-- [[[bib14,14]]]  Gavin King. Context and Dependency Injection for the Java Platform. JSR, JCP, December 2009.
-               See https://jcp.org/en/jsr/detail?id=299.
+- [[[bib14,14]]]  Jakarta™ Contexts and Dependency Injection Specification, Version 2.0. Available at:
+               https://jakarta.ee/specifications/cdi/2.0
 
-- [[[bib15,15]]]  Rajiv Mordani. Common Annotations for the Java Platform. JSR, JCP, July 2005. See
-               https://jcp.org/en/jsr/detail?id=250.
+- [[[bib15,15]]]  Jakarta™ Annotations Specification, Version 1.3. Available at:
+               https://jakarta.ee/specifications/annotations/1.3
 
-- [[[bib16,16]]]  Emmanuel Bernard. Bean Validation 1.1. JSR, JCP, March 2013. See
-               https://jcp.org/en/jsr/detail?id=349.
+- [[[bib16,16]]]  Jakarta™ Bean Validation Specification, Version 2.0. Available at:
+               https://jakarta.ee/specifications/bean-validation/2.0
 
 - [[[bib17,17]]]  Server-Sent Events. See https://html.spec.whatwg.org/#server-sent-events.
 
-- [[[bib18,18]]]  Dmitry Kornilov. Java API for JSON Processing 1.1. JSR, JCP, May 2017. See
-               https://jcp.org/en/jsr/detail?id=374.
+- [[[bib18,18]]]  Jakarta™ JSON Processing Specification, Version 1.1. Available at:
+               https://jakarta.ee/specifications/jsonp/1.1
 
-- [[[bib19,19]]]  Dmitry Kornilov. Java API for JSON Binding. JSR, JCP, July 2017. See
-               https://jcp.org/en/jsr/detail?id=367.
+- [[[bib19,19]]]  Jakarta™ JSON Binding Specification, Version 1.0. Available at:
+               https://jakarta.ee/specifications/jsonb/1.0
 
-- [[[bib20,20]]]  Bill Shannon. JavaBeans Activation Framework. JSR, JCP, May 2006. See
-               https://jcp.org/en/jsr/detail?id=925.
\ No newline at end of file
+- [[[bib20,20]]]  JavaBeans™ Activation Framework Specification Version 1.2 (JAF specification). Available at:
+               https://www.oracle.com/technetwork/java/javase/tech/index-jsp-138795.html
```

Comment:
This link doesn't work for me - it just takes me to the Java SE technologies page, and I cannot find any good info on JAF there.  For this one, we might want to leave it as the JSR link until JAF is moved to the Jakarta community.

Reply:
The migration of Activation to Jakarta is decided, but has not started. Hence there yet is not such URL and I doubt it won't change in near future. So it would be best to keep to old one (as it is still correct) until the new one is finally defined.